smaug/python/ops/common.py

```diff
@@ -81,35 +103,35 @@ def broadcast_inputs(tensor_a, tensor_b, name="broadcast_inputs"):
   operations.
 
   Args:
-    tensor_a, tensor_b: Two input tensors.
+    tensor_a: The first input tensor.
+    tensor_b: The second input tensor.
     name: Name prefix for the operators used in this function.
 
   Returns:
     Two new tensors with the same shape.
 
   Examples:
 
-  ```python
-  a = np.random.rand(2, 8).astype(np.float16)
-  b = np.random.rand(2, 1).astype(np.float16)
-  tensor_a = Tensor(data_layout=NC, tensor_data=a)
-  tensor_b = Tensor(data_layout=NC, tensor_data=b)
-  # The elementwise add operator calls _broadcast_inputs() so that tensor_b is
-  # broadcast in axis 1, making both inputs shaped [2, 8].
-  output = add(tensor_a, tensor_b)
-  ```
-
-  ```python
-  a = np.random.rand(2, 16, 1, 8).astype(np.float16)
-  b = np.random.rand(2, 1, 8, 8).astype(np.float16)
-  tensor_a = Tensor(data_layout=NHWC, tensor_data=a)
-  tensor_b = Tensor(data_layout=NHWC, tensor_data=b)
-  # The elementwise mul operator calls _broadcast_inputs() so that both inputs
-  # will be shaped [2, 16, 8, 8].
-  output = mul(tensor_a, tensor_b)
-  ```
+  .. code:: python
```

Comment:
So I guess using ```python doesn't generate what we want here?

Reply:
Right.

Reply:
Is `code-block` actually the same as `code`?

Reply:
code-block is equivalent to three backticks in Markdown; code is equivalent to one backtick.